pages/task/list/components/assign-task.jsx

```diff
@@ -44,7 +44,7 @@ export default function AssignTask({ task }) {
 
   return (
     <Fragment>
-      <em>Unassigned</em>
+      <em className='block'>Unassigned</em>
```

Comment:
I'm a little surprised that passed linting.
```suggestion
      <em className="block">Unassigned</em>
```

Reply:
Sorted